bim2sim/utilities/visualize_spaces.py

```diff
@@ -1,45 +0,0 @@
-import ifcopenshell
```

Comment:
keep this file and replace with plot function which you just tried to move to TEASER Plugin.